Makefile

```diff
@@ -1668,7 +1665,7 @@ $(OBJDIR)/%/build/riscv-gnu-toolchain/build-picolibc/stamp: \
 	      -Dsysroot-install=true \
 	      --prefix $(abspath $($@_INSTALL))/$(NEWLIB_TUPLE) \
 	      --cross-file $(SRC_PICOLIBC)/cross-riscv64-unknown-elf.txt
-	$(NINJA) -C $(dir $@) install >& $(dir $@)/make-install.log
+	$(NINJA) -C $(dir $@) -j1 install >& $(dir $@)/make-install.log
```

Comment:
This is Ninja and not Make, so I think that the -j1 flag should be removed here.
It has worked before without it, so I am not sure how it will work with it.
After removing this, I think you should start a build across all platforms on the Jenkins CI, just to check that it works as expected.

Reply:
Good catch!
Thanks!